plugins/retrolab/fps_retrolab/routes.py

```diff
@@ -337,7 +339,7 @@ def get_index(doc_name, retro_page, collaborative, base_url="/"):
   <script id="jupyter-config-data" type="application/json">
     PAGE_CONFIG
   </script>
-  <script src="/static/retro/bundle.js" main="index"></script>
+  <script src="BASE_URLstatic/retro/bundle.js" main="index"></script>
```

Comment:
Maybe it could be useful to follow a similar syntax as in other template engines, for example with the `{{ BASE_URL  }}`?

Reply:
Yes, it's very fragile at the moment. But then without needing Jinja, right?

Reply:
> But then without needing Jinja, right?

Yes still without Jinja. Just so it sticks out when reading the code.